spec/forms/location_form_spec.rb

```diff
@@ -161,4 +161,15 @@
       expect(location.children.count).to eq(1)
     end
   end
+
```

Comment:
I am wondering whether this test sits better in the ordered location model?

Reply:
I've also added a test in there 👍